sdk/highlight-cloudflare/src/sdk.ts

```diff
@@ -63,8 +63,17 @@ export const H: HighlightInterface = {
 				['highlight.trace_id']: requestID,
 			}),
 		})
+
 		for (const m of RECORDED_CONSOLE_METHODS) {
-			console[m] = sdk.logger[m]
+			const originalConsoleMethod = console[m]
+
+			console[m] = (message: string, ...args: unknown[]) => {
```

Comment:
I noticed that once I'd called `H.init`, I never saw anything in my console, ever again.

This logic preserves the original console methods and calls them alongside the `sdk.logger`.